docs/integrations/sources/confluence.md

```diff
@@ -1,22 +1,35 @@
 # Confluence
 
-This page contains the setup guide and reference information for the Confluence source connector.
+This page contains the setup guide and reference information for the 
+[Confluence](https://www.atlassian.com/software/confluence) source connector.
 
 ## Prerequisites
 
-* [API Token](https://support.atlassian.com/atlassian-account/docs/manage-api-tokens-for-your-atlassian-account/)
+* Atlassian API Token
 * Your Confluence domain name
 * Your Confluence login email
 
 ## Setup guide
-### Step 1: Set up Confluence connector
-1. Log into your [Airbyte Cloud](https://cloud.airbyte.io/workspaces) or Airbyte Open Source account.
-2. Click **Sources** and then click **+ New source**. 
-3. On the Set up the source page, select **Confluence** from the Source type dropdown.
-4. Enter a name for your source.
-5. For **API Token** follow the Jira confluence for generating an  [API Token](https://support.atlassian.com/atlassian-account/docs/manage-api-tokens-for-your-atlassian-account/)
-6. For **Domain name** enter your Confluence domain name.
-7. For **Email** enter your Confluence login email.
+### Create an API Token 
+
+For detailed instructions on creating an Atlassian API Token, please refer to the 
+[official documentation](https://support.atlassian.com/atlassian-account/docs/manage-api-tokens-for-your-atlassian-account/).
+
+### Set up Confluence connector in Airbyte
+1. [Log in to your Airbyte Cloud](https://cloud.airbyte.com/workspaces) account, or navigate to the Airbyte Open Source dashboard.
+2. In the left navigation bar, click **Sources**.
+
+:::tip
```

Comment:
You didn't originate this, but I think we can improve on these 4 steps. They take up a lot of visual space (and since we show these docs in the app itself, they're a bit redundant). I'd rather we collapsed them into 1 step e.g `From the Airbyte UI, click **Sources** then click on **New Source** and select **Confluence**`

Including removing this `tip`

Reply:
Sounds good! Collapsed as suggested 👍